tox.ini

```diff
@@ -5,7 +5,6 @@ envlist =
 [testenv]
```

Comment:
Is `tox` even used?  I can remove this file if not.

Reply:
Not by Travis, but I've been using it on my computer.

Reply:
Switched to Python 2.7 and 3.6.